pkgs/applications/networking/cluster/docker-machine-kvm/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildGoPackage, fetchFromGitHub, libvirt, libvirt-glib, kvm, docker-machine }:
+
+buildGoPackage rec {
+    name = "docker-machine-kvm-${version}";
+    version = "20160801-${stdenv.lib.strings.substring 0 7 rev}";
```

Comment:
I'll try. I had some troubles with v0.7.0. Last updates (few) fix it (I think). If I have same issues with v0.7, I'll ask maintainer to tag master.